boards/arm/adafruit_trinket_m0/doc/adafruit_trinket_m0.rst

```diff
@@ -0,0 +1,147 @@
+.. _adafruit_trinket_m0:
+
+Adafruit Tinket M0
+##################
+
+Overview
+********
+
+The Adafruit Trinket M0 is a tiny (27 mm x 15 mm) development board
+with an onboard RGB LED, USB port, and range of I/O broken out onto 5
+pins.
+
+.. image:: img/adafruit_trinket_m0.png
+     :width: 500px
+     :align: center
+     :alt: Adafruit Trinket M0
+
+Hardware
+********
+
+- ATSAMD21E18A ARM Cortex-M0+ processor at 48 MHz
+- 256 KiB flash memory and 32 KiB of RAM
+- Internal trimmed 8 MHz oscillator
+- A user LED
+- A RGB DotStar LED
```

Comment:
``An RGB``   (and yes, English rules for using ``a`` vs. ``an`` are tricky, based on the sound of the next word)

Reply:
Done